configure.ac

```diff
@@ -717,13 +726,20 @@ AC_SUBST(OLA_CLIENT_LIBS)
 
 # Extra tools
 #####################################################
-AM_CONDITIONAL([BUILD_JA_RULE],
-               [test "x$enable_ja_rule" = xyes && test "x$have_libusb" = xyes \
-                && test "x$have_libusb_hotplug_api" = xyes])
+AS_IF([test "x$enable_ja_rule" = xyes && test "x$enable_usbdmx" != xyes],
```

Comment:
It probably makes sense to check for != xno for ja_rule and = xno for usbdmx to support a future auto mode.